sklearn/cluster/tests/test_k_means.py

```diff
@@ -53,18 +53,25 @@
 X, true_labels = make_blobs(
     n_samples=n_samples, centers=centers, cluster_std=1.0, random_state=42
 )
-X_csr = sp.csr_matrix(X)
+X_csr_containers = [container(X) for container in CSR_CONTAINERS]
```

Comment:
I find it weird to use the "container" name both for the instance and the type of container.

Maybe renaming this to `X_as_any_csr` would avoid introducing this confusion.

Reply:
Do I understand right: I have to change name of variable `X_csr_containers` -> `X_as_any_csr`.